python/lsst/sdm_schemas/schemas/imsim.yaml

```diff
@@ -6015,7 +6015,608 @@ tables:
+    fits:tunit:
+  - name: z_psfModel_TwoGaussian_is_parent_flag
+    "@id": "#Object.z_psfModel_TwoGaussian_is_parent_flag"
+    datatype: boolean
+    description: Flag set for objects not fit because they are parents for the two-Gaussian PSF model (z-band).
+    fits:tunit:
+      # MultiProFit Sersic model
+  - name: sersic_cen_x
+    "@id": "#Object.sersic_cen_x"
+    datatype: double
+    description: Centroid (tract, x-axis) from the multiband Sersic model fit.
+    fits:tunit: pixel
+  - name: sersic_cen_y
+    "@id": "#Object.sersic_cen_y"
+    datatype: double
+    description: Centroid (tract, y-axis) from the multiband Sersic model fit.
+    fits:tunit: pixel
+  - name: sersic_cen_ra
+    "@id": "#Object.sersic_cen_ra"
+    datatype: double
+    description: Centroid (right ascension) from the multiband Sersic model fit.
+    fits:tunit: deg
+  - name: sersic_cen_dec
+    "@id": "#Object.sersic_cen_dec"
+    datatype: double
+    description: Centroid (declination) from the multiband Sersic model fit.
+    fits:tunit: deg
+  - name: sersic_reff_x
+    "@id": "#Object.sersic_reff_x"
+    datatype: double
+    description: Effective radius (x-axis) from the multiband Sersic model fit, prior to convolution with the adjusted (see config) PSF value.
+    fits:tunit: pixel
+  - name: sersic_reff_y
+    "@id": "#Object.sersic_reff_y"
+    datatype: double
+    description: Effective radius (y-axis) from the multiband Sersic model fit, prior to convolution with the adjusted (see config) PSF value.
+    fits:tunit: pixel
+  - name: sersic_rho
+    "@id": "#Object.sersic_rho"
+    datatype: double
+    description: Ellipse rho (correlation coefficient) from the multiband Sersic model fit.
+    fits:tunit:
+  - name: g_sersicFlux
+    "@id": "#Object.g_sersicFlux"
+    datatype: double
+    description: g-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: i_sersicFlux
+    "@id": "#Object.i_sersicFlux"
+    datatype: double
+    description: i-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: r_sersicFlux
+    "@id": "#Object.r_sersicFlux"
+    datatype: double
+    description: r-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: u_sersicFlux
+    "@id": "#Object.u_sersicFlux"
+    datatype: double
+    description: u-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: y_sersicFlux
+    "@id": "#Object.y_sersicFlux"
+    datatype: double
+    description: y-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: z_sersicFlux
+    "@id": "#Object.z_sersicFlux"
+    datatype: double
+    description: z-band flux from the multiband Sersic model fit.
+    fits:tunit: nJy
+  - name: sersic_sersic_index
+    "@id": "#Object.sersic_sersic_index"
+    datatype: double
+    description: Sersic profile index parameter from the multiband Sersic model fit.
+    fits:tunit:
+  - name: sersic_cen_xErr
```

Comment:
I recommend dropping these pixel-coordinate uncertainties along with the pixel-coordinate values.